js/piwik.js

```diff
@@ -4701,6 +4729,18 @@ if (typeof Piwik !== 'object') {
                     configRequestMethod = method || defaultRequestMethod;
                 },
 
+                activeBulkTracking: function(active) {
```

Comment:
Just for consistency: So far we use `enableLinkTracking`, `enableHeartBeatTimer`, `enableTrackOnlyVisibleContent`, `disableHeartBeatTimer`.

Maybe there could be a `enableBulkTracking` and `disableBulkTracking` instead. There's nothing wrong the way you did it, it would be only for consistency